spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: recipes
+#
+#  id          :bigint           not null, primary key
+#  name        :string
+#  description :text
+#  ingredients :text
+#  user_id     :bigint           not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_recipes_on_user_id  (user_id)
+#
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Remember model spec is for model validations and associations, so something simple like this could go here:
```suggestion
  describe 'Validations' do
    it { is_expected.to validate_presence_of(:ingredients) }
  end

  describe 'Associations' do
    it { is_expected.to belong_to(:user) }
  end
```